resources/responses/skroutz/merchants_ecommerce_orders_demo_rejected.yml

```diff
@@ -0,0 +1,89 @@
+---
+:request:
+  :url: https://api.skroutz.gr/merchants/ecommerce/orders/DEMO-REJECTED
+  :headers:
+    :accept: application/vnd.skroutz+json; version=3.12
```

Comment:
I don't think the request headers are shown, but -for Orders API- we use the 3.0 version :) We may adjust this

Reply:
Okay! The request headers can be seen by Show Response -> Show Headers, so yes I will fix it